pypi_formula_mappings.json

```diff
@@ -550,8 +550,8 @@
   "img2pdf": {
     "exclude_packages": ["lxml", "packaging", "pillow"]
   },
-  "instalooter": {
-    "exclude_packages": ["certifi", "six"]
+  "instaloader": {
+    "exclude_packages": ["requests"]
```

Comment:
This should be changed back.

Reply:
aha, I was using a wrong one, let me fix it.